src/test/java/de/fraunhofer/aisec/cpg/TestUtils.java

```diff
@@ -59,30 +62,68 @@
     return nodes.stream().filter(m -> m.getName().equals(name)).collect(Collectors.toList());
   }
 
-  public static List<TranslationUnitDeclaration> analyze(String fileExtension, Path topLevel)
+  /**
+   * Like {@link #analyze(List, Path, boolean)}, but for all files in a directory tree having a
+   * specific file extension
+   *
+   * @param fileExtension All files found in the directory must end on this String. An empty string
+   *     matches all files
+   * @param topLevel The directory to traverse while looking for files to parse
+   * @param usePasses Whether the analysis should run passes after the initial phase
+   * @param cleanupOnCompletion Whether {@link de.fraunhofer.aisec.cpg.graph.TypeManager} etc.
+   *     should be cleaned up after the analysis has ended
+   * @return A list of {@link TranslationUnitDeclaration} nodes, representing the CPG roots
+   * @throws Exception Any exception thrown during the parsing process
+   */
+  public static List<TranslationUnitDeclaration> analyze(
+      String fileExtension, Path topLevel, boolean usePasses, boolean cleanupOnCompletion)
```

Comment:
```suggestion
      String fileExtension, Path topLevel, boolean usePasses)
```

Reply:
Huh seems like I forgot these

Reply:
:) Almost there. Nice job with mockito.

Reply:
Ah.. you need to get rid of the additional parameters of the `analyze` call as well ;)

Reply:
:sweat_smile: